recipes-debian/libcheck/libcheck_debian.bb

```diff
@@ -0,0 +1,34 @@
+# base recipe: recipes-support/libcheck/libcheck_0.12.0.bb
+# base branch: warrior
+# base commit: 21360dd971b39306b0332212a36ba9211f66e3a1
+
+SUMMARY  = "Check - unit testing framework for C code"
+HOMEPAGE = "https://libcheck.github.io/check/"
+SECTION = "devel"
+
+LICENSE  = "LGPLv2.1+"
+LIC_FILES_CHKSUM = "file://COPYING.LESSER;md5=2d5025d4aa3495befef8f17206a5b0a1"
+
+inherit debian-package
```

Comment:
@mlhs19 line 12,13 and line 25,26 are duplicated. Please delete ether.

Reply:
I removed 25,26. force pushed.

Reply:
> I removed 25,26. force pushed.

@mlhs19 It seems your push https://github.com/miraclelinux/meta-debian-extended/commit/392a7ee46c7297145fe0c12429ca9747b8b4badc didn't remove line 25,26. Would you check it please?

Reply:
Sorry. I forgot to add. force pushed as 29bcf16

Reply:
thanks